libs/storage/Tsavorite/cs/test/TestUtils.cs

```diff
@@ -134,14 +129,14 @@ internal static IDevice CreateTestDevice(DeviceType testDeviceType, string filen
 
             switch (testDeviceType)
             {
-#if WINDOWS
-                case DeviceType.LSD:
+                case DeviceType.LSD when !OperatingSystem.IsWindows():
+                    Assert.Inconclusive();
```

Comment:
same comment as above. This might be better as an Assert.Fail, since we shouldn't be asking for LSD specifically on non-Windows.

Reply:
I removed the preprocessor which previously were on the DeviceType enum. Is following better or do I revert last commit entirely?
```suggestion
                    Assert.Ignore($"Skipping {nameof(DeviceType.LSD)} on non-windows");
```

Reply:
Better to make this Assert,Fail; the test needs attention if it's requesting LSD on non-Windows. It returned  null device before in that case, so there'll be no new break.

Reply:
Ok 👍 I'll check all `DeviceType` enums to not include LSD

Reply:
The change above this method removed the #ifdef from the LSD enum definition, so I fear we can't make this fail 🤔

Reply:
Oh, right--[Values] runs them all. Ignore() it is, then.